packages/wallet/src/redux/reducer.ts

```diff
@@ -90,3 +106,25 @@ const waitForLoginReducer = (
       return state;
   }
 };
+function startProcess(
+  state: states.Initialized,
+  sharedData: states.SharedData,
+  action: {
+    type: 'WALLET.INDIRECT_FUNDING.FUNDING_REQUESTED';
+    channelId: string;
+    playerIndex: PlayerIndex;
+    protocol: WalletProtocol;
+  },
+  protocolState:
+    | states.indirectFunding.playerA.WaitForApproval
```

Comment:
Could we type this more broadly? Like `ProtocolState`?

It seems strange to have a `startProcess` method expecting  just `IndirectFunding` arguments. It seems like it should take `ProtocolState` and a `ProtocolAction` even if thats only happens to be indirect funding stuff right now.

Reply:
Yeah, thanks for catching that. It's from a VSCode refactor that I neglected to update.